interface/patient_file/summary/immunizations.php

```diff
@@ -414,7 +414,7 @@ function saveImmunizationObservationResults($id, $immunizationdata)
                     </div>
                     <div class="form-group mt-3">
                         <label><?php echo xlt('Immunization Manufacturer'); ?></label>
-                        <?php echo generate_select_list('manufacturer', 'Immunization_Manufacturer', ($manufacturer ?? ''), 'Select Manufacturer', '');?>
+                        <?php echo generate_select_list('manufacturer', 'Immunization_Manufacturer', ($manufacturer ?? ''), 'Select Manufacturer', ' ');?>
```

Comment:
odd that need this space here. does this fix something?

Reply:
the space allows the manufacturer to not populate with abbot laboratories by default when adding an imm

Reply:
makes sense :)